test/plugin/test_in_forward.rb

```diff
@@ -465,6 +469,68 @@ def test_skip_invalid_event(data)
     end
   end
 
+  class CompressedPackedForward < self
+    extend Fluent::Test::StartupShutdown
+
+    def test_set_compress_to_option
+      d = create_driver
+
+      time = Fluent::EventTime.parse("2011-01-02 13:14:15 UTC").to_i
+      events = [
+        ["tag1", time, {"a"=>1}],
+        ["tag1", time, {"a"=>2}]
+      ]
+
+      # create compressed entries
+      entries = ''
+      events.each do |_tag, _time, record|
+        v = ''
+        [_time, record].to_msgpack(v)
```

Comment:
It's already deprecated feature to assign String objects as an argument of `#to_msgpack`.
Please update the code as `v << [_time, record].to_msgpack`.